packages/shared/components/atoms/MediaDisplay.svelte

```diff
@@ -0,0 +1,104 @@
+<script lang="typescript">
+    import { MimeType, ParentMimeType } from '@core/nfts'
+
+    export let Media: HTMLImageElement | HTMLVideoElement
+    export let src: string
+    export let expectedType: MimeType
+    export let classes: string
+    export let alt
```

Comment:
is this required? if not, add ` = undefined` or `= ''`

Reply:
```suggestion
    export let alt = ''
```